.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__/
 .env
-env/
\ No newline at end of file
+env/
+docker-compose.yaml
```

Comment:
This file should not be ignored. Curious though why you are ignoring it?

Reply:
I created two docker files. In the ignored file, under environment, I provide database connection details. This is to ensure that I don't push my docker-compose file with secret information

Reply:
@APIYOJENNIFER, we typically do not ignore the compose files. Those are the heart of docker images. Instead, what you can do, is have a `.env` which [contains](https://docs.docker.com/compose/environment-variables/set-environment-variables/) the values. In that case, add an extra `.env.dev` file that contains data you are to use for development. This should also guide what values are expected in the .env file. You may add the instructions in the readme.